altislife.sql

```diff
@@ -27,43 +29,49 @@ DROP PROCEDURE IF EXISTS `deleteOldHouses`;
 DROP PROCEDURE IF EXISTS `deleteOldGangs`;
 DROP PROCEDURE IF EXISTS `deleteOldContainers`;
 DROP PROCEDURE IF EXISTS `deleteOldWanted`;
+DROP PROCEDURE IF EXISTS `noop`;
 
 DELIMITER $$
 --
 -- Procedures
 -- CURRENT_USER function returns the name of the current user in the SQL Server database.
 --
 
-CREATE DEFINER=CURRENT_USER PROCEDURE `resetLifeVehicles`()
+CREATE DEFINER=`arma3`@`localhost` PROCEDURE `resetLifeVehicles`()
 BEGIN
   UPDATE `vehicles` SET `active`= 0;
 END$$
 
-CREATE DEFINER=CURRENT_USER PROCEDURE `deleteDeadVehicles`()
+CREATE DEFINER=`arma3`@`localhost` PROCEDURE `deleteDeadVehicles`()
 BEGIN
   DELETE FROM `vehicles` WHERE `alive` = 0;
 END$$
 
-CREATE DEFINER=CURRENT_USER PROCEDURE `deleteOldHouses`()
+CREATE DEFINER=`arma3`@`localhost` PROCEDURE `deleteOldHouses`()
 BEGIN
   DELETE FROM `houses` WHERE `owned` = 0;
 END$$
 
-CREATE DEFINER=CURRENT_USER PROCEDURE `deleteOldGangs`()
+CREATE DEFINER=`arma3`@`localhost` PROCEDURE `deleteOldGangs`()
 BEGIN
   DELETE FROM `gangs` WHERE `active` = 0;
 END$$
 
-CREATE DEFINER=CURRENT_USER PROCEDURE `deleteOldContainers`()
+CREATE DEFINER=`arma3`@`localhost` PROCEDURE `deleteOldContainers`()
 BEGIN
   DELETE FROM `containers` WHERE `owned` = 0;
 END$$
 
-CREATE DEFINER=CURRENT_USER PROCEDURE `deleteOldWanted`()
+CREATE DEFINER=`arma3`@`localhost` PROCEDURE `deleteOldWanted`()
```

Comment:
Please keep this is CURRENT_USER

Reply:
don'T know much about current-user - but when replacing them via phpMyAdmin it defaults them to the logged in user - hence my manual move of creating the user to the very top

about user not have delete permissions: makes it unnecessary harder to correctly administrate it for cleaning out junk to switch to another user with more privileges (root)
also a database wipe using TRUNCATE is impossible with the limited rights as most implementations (at least MySQL and MariaDB) just completely drop the table and create a new one instead of clearing it out

Reply:
https://github.com/AsYetUntitled/Framework/issues/633

Reply:
The point of the CURRENT_USER is exactly as stated above. The procedure would be ran as the user who runs the SQL script (one with elevated permissions) and **not** the Arma3 user. This gives an additional layer of security as the "elevated" user is never used at the application to login to the database, and instead a user with less permissions is used instead.

Would be great if you could change this back as previously stated, thanks!

Reply:
> Would be great if you could change this back as previously stated, thanks!

just deny the PR
as said: it's just a show of to show what can be fixed by ONE dev and two accounts (picked up a 2nd copy as it was on sale) within a couple of hours on a sunday afternoon what this whole repo isn'T able to get done within months ...
it's also a bit uncool that someone thought to re-open the closed issues just cause s/he couldn't take a "NO, that's it, I'm pulling out" - it's still MY stuff on which I have the copyright - if I decide to close it as I don't see it anymore relevant as it's either denied by someone with at least in my eyes lacks a lot of common knowledge or noone even gives a damn to at least look at it - than that's my decision. Only stupid rights management of GitHub allows you as maintainers to fiddle with other users content ... which I see as a major flaw of githubs rights management ...

anyway - theese changes are just provided to get v6 movin a bit - do withit what you want - use it - shred it - IDK - but at least keep the rolling train movin instead of let get it stuck again im limbo for the next months ...
I already started a re-write from scratch - and instead of just supporting a HC my mission will rely on it - as it really decreases system resource consumption and should be used by anyone - even on a windows host (ok, anyone using windows as server os seem to have other issues to solve) with 64bit support - it'S at least somewhat useful unless BI provides a 64bit linux binary

so long suxxers - feel free to block as this account is most likely already deleted when you reply
just thought most admins were morrons - but seeing how this whole project is managed - well, I can see where at least SOME got their attitude from ..

Reply:
> within a couple of hours on a sunday afternoon what this whole repo isn'T able to get done within months

You should bare in mind that people don't have to commit all of their free time (some of who have less than others) to this project. Considering the current worldwide situation I wouldn't be surprised if people have other priorities.

> it's also a bit uncool that someone thought to re-open the closed issues

The issues are still valid, you've reported a problem with the code base. The code you submitted to be merged is tied to your pull request. Whether or not your PR is merged has nothing to do with the state of the issue.

> I already started a re-write from scratch

That's great! I look forward to seeing it :) best of luck